dir.c

```diff
@@ -3689,7 +3689,7 @@ Init_Dir(void)
      * {File::FNM_SYSCASE}[rdoc-ref:File::Constants@File-3A-3AFNM_SYSCASE] */
     rb_file_const("FNM_SYSCASE", INT2FIX(FNM_SYSCASE));
     /* Document-const: FNM_SHORTNAME
-     * {File::FNM_SHORTNAME}[rdoc-ref:File::Constants@File-3A-3AFNM_SHORTNAME] */
+     * {File::FNM_SHORTNAME}[rdoc-ref:File::Constants@File-3A-3AFNM_SHORTNAME+-28Windows+Only-29] */
```

Comment:
I'd rather propose moving "(Windows Only)" from the heading to its content.

```
     * ==== File::FNM_SHORTNAME
     *
     * Flag File::FNM_SHORTNAME Allows patterns to match short names if they exist.
     * (Windows Only).
```

Reply:
Heading and link fixed, along with one other similar header and its link.